Include/XA/kzma/mynn.sid

```diff
@@ -1,7 +1,7 @@
 //MYNN 14 FREDY2 NICKO
 MYNN;14;FREDY2;INURE;INURE;
 FREDY;FREDY;
-JUTED;JUTED;
+JUTED;JUTED;Max 210K

```

Comment:
Normally this should be written as `JUTED;JUTED;v210K;` Missing semicolon will break the following fix as well.